<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBoard</class>
 <widget class="QDialog" name="FormBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textID">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>104</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textColor">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>224</y>
     <width>104</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textWidth">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>257</y>
     <width>104</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textHeight">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>104</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
